docs/version-1.9/modules/en/pages/upgrades/longhorn-components/upgrade-longhorn-manager.adoc

```diff
@@ -3,27 +3,33 @@
 
 == Upgrading from v1.8.x
 
+[NOTE]
+====
+It is recommended to create a xref:snapshots-backups/system-backups/create-system-backup.adoc[Longhorn system backup] before performing the upgrade. This ensures that all critical resources, such as volumes and backing images, are backed up and can be restored in case any issues arise.
+====
+
```

Comment:
Can we use `Notice`?
```suggestion
[NOTE]
====
It is recommended to create a xref:snapshots-backups/system-backups/create-system-backup.adoc[Longhorn system backup] before performing the upgrade. This ensures that all critical resources, such as volumes and backing images, are backed up and can be restored in case any issues arise.
====

== Upgrading from v1.8.x
```